src/soundio/sounddevicenetwork.cpp

```diff
@@ -443,8 +443,32 @@ void SoundDeviceNetwork::callbackProcessClkRef() {
         } else {
             qDebug() << "SSE: Denormals to zero mode is working";
         }
+#elif defined(aarch64)
+        // Flush-to-zero on aarch64 is controlled by the Floating-point Control Register
+        // Load the register into our variable.
+        int savedFPCR;
+        asm volatile("mrs %[savedFPCR], FPCR"
+                     : [ savedFPCR ] "=r"(savedFPCR));
+
+        qDebug() << "aarch64 FPCR: setting bit 24 to 1 to enable Flush-to-zero";
+        // Bit 24 is the flush-to-zero mode control bit. Setting it to 1 flushes denormals to 0.
+        asm volatile("msr FPCR, %[src]"
+                     :
+                     : [ src ] "r"(savedFPCR | (1 << 24)));
+
+        // verify if flush to zero or denormals to zero works
+        // test passes if one of the two flag is set.
+        volatile double doubleMin = DBL_MIN; // the smallest normalized double
+        VERIFY_OR_DEBUG_ASSERT(doubleMin / 2 == 0.0) {
+            qWarning() << "aarch64 FPCR: Flush-to-zero mode is not working. "
+                          "EQs and effects may suffer high CPU load";
+        }
+        else {
+            qDebug() << "aarch64 FPCR: Flush-to-zero mode is working";
+        }
 #else
-        qWarning() << "No SSE: No denormals to zero mode available. EQs and effects may suffer high CPU load";
+        qWarning() << "No SSE or aarch64 FPCR: No denormals to zero mode "
```

Comment:
This is not true in case of arm32. In case of Neon floating point unit Flush to zero is always enabled. 
Can you use here a common  VERIFY_OR_DEBUG_ASSERT(doubleMin / 2 == 0.0) { block like in sounddeviceportaudio.cpp?

Reply:
Very well observed @daschuer I had missed that. Will work on it.